public/locales/es-ES/grafana.json

```diff
@@ -49,19 +49,14 @@
     },
     "counts": {
       "alertRule_one": "{{count}} regla de alerta",
-      "alertRule_many": "",
```

Comment:
Why is this deletion just happening in Spanish and French? Shouldn't it be in sync? I mean, as we are deleting the translation keys, should this not be the same in all languages?

Reply:
Crowdin says that spanish only has singular and other plural forms, hence why the `_many` is being removed here.


![image](https://github.com/grafana/grafana/assets/46142/7972e8f7-cbc7-4764-a8f2-c6337453238c)

Reply:
Thanks Josh for the explanation 👌